wpimath/src/main/native/include/frc/controller/SimpleMotorFeedforward.h

```diff
@@ -77,82 +84,32 @@ class SimpleMotorFeedforward {
   }
 
   /**
-   * Calculates the feedforward from the gains and setpoint.
-   * Use this method when the setpoint does not change.
+   * Calculates the feedforward from the gains and setpoint assuming discrete
```

Comment:
The velocity-acceleration overload should say "assuming continuous control" for symmetry.

Reply:
comments added